### PR TITLE
Update circuits doc

### PR DESCRIPTION
**Pull request by vivianjeng**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally in root directory and any changes were pushed
- [x] Lint (`yarn lint:fix`) was run locally in root directory and any changes were pushed

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/docs/docs/circuits-api/circuits.md [hunk 1]
```diff
@@ -22,7 +23,7 @@ import { Circuit } from '@unirep/circuits'
 
 ## Signup Proof
 
-The signup proof outputs a state tree leaf and an identity commitment for the user. The state tree leaf will have zero values for all data fields.
+The signup proof outputs a [state tree](../protocol/trees.md#state-tree) leaf and an [identity commitment](https://semaphore.appliedzkp.org/docs/glossary#identity-commitment) for the user. The state tree leaf will have zero values for all data fields.
 
 Control field:
```
Comment (CJ-Rose) [inline packages/docs/docs/circuits-api/circuits.md:28]: there is a typo in each of the blue info boxes (4), it should read:
"Control fields are 'used' to encode.."

File: packages/docs/docs/circuits-api/circuits.md [hunk 2]
```diff
@@ -158,7 +159,7 @@ Outputs:
 - `control[2]`
 
 :::info
-Control fields are use to encode many small values into a single field element. This reduces the number of public signals needed to operate a circuit.
+Control fields are used to encode many small values into a single field element. This reduces the number of public signals needed to operate a circuit.
 :::
 
 ## User State Transition Proof
```
Comment (CJ-Rose) [inline packages/docs/docs/circuits-api/circuits.md:165]: do you mean to leave the 'TODO' on line 171?

File: packages/docs/docs/circuits-api/circuit-config.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: CircuitConfig
+---
+
+Use the default circuit config like so
```
Comment (CJ-Rose) [inline packages/docs/docs/circuits-api/circuit-config.md:5]: add : to end of line

File: packages/docs/docs/circuits-api/installation.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Installation"
+---
+
+`@unirep/circuits` is a client library for circuit related functions which are used in unirep protocol.
```
Comment (CJ-Rose) [inline packages/docs/docs/circuits-api/installation.md:5]: capitalize UniRep

File: packages/docs/docs/circuits-api/base-proof.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ constructor(
```
Comment (CJ-Rose) [inline packages/docs/docs/circuits-api/base-proof.md:41]: on line 76:
- remove comma after 'constructor' and replace it after 'manually'
- should `Prover` and `prover` both be capitalized or lowercase?

**Review by CJ-Rose — COMMENTED** [review]
another small edit in docs/circuits-api/epoch-key-proof.md:
on line 25 change:
'should be verified to exist onchain. When verifying the proof.'
to
'should be verified to exist onchain when verifying the proof.'

File: packages/docs/docs/circuits-api/base-proof.md (hunk 5 above)
Comment (vivianjeng) [reply packages/docs/docs/circuits-api/base-proof.md:41]: `Prover` is an interface: https://github.com/Unirep/Unirep/blob/81bb7605b3f7fd215ca15f05c9cdc6c119841a83/packages/circuits/src/circuits.ts#L19
and `prover` is an object with the `Prover` interface.
Is the description correct here?

File: packages/docs/docs/circuits-api/installation.md (hunk 4 above)
Comment (vivianjeng) [reply packages/docs/docs/circuits-api/installation.md:5]: fixed in PR #509

File: packages/docs/docs/circuits-api/circuits.md (hunk 2 above)
Comment (vivianjeng) [reply packages/docs/docs/circuits-api/circuits.md:165]: commented out the TODO in the document

**Review by CJ-Rose — APPROVED** [review]

**Pull request merged**